tests/test_feature_manager.cc

```diff
@@ -34,11 +34,19 @@ namespace rascal {
   /* ---------------------------------------------------------------------- */
   // TODO(felix) define more test that could be streamlined
   // gets a list of fixtures for all the different possible structure managers
-  using multiple_fixtures = boost::mpl::list<
-      FeatureFixture<double, FeatureManagerDense, StructureManagerCenters,
-                     RepresentationManagerSortedCoulomb, TestFeatureData>,
-      FeatureFixture<float, FeatureManagerDense, StructureManagerCenters,
-                     RepresentationManagerSortedCoulomb, TestFeatureData>>;
+  using multiple_fixtures =
+      boost::mpl::list<FeatureFixture<double, FeatureManagerDense,
+                                      RepresentationManagerSortedCoulomb,
+                                      MultipleStructureSortedCoulomb>,
+                       FeatureFixture<float, FeatureManagerDense,
```

Comment:
I am not following, what is done here - is this just a test to see if a list can be constructed?

Reply:
Yes and also does not change the data it is feed. It will change drastically after the changes to the structure manager aggregator